images/php-fpm/7.4.Dockerfile

```diff
@@ -73,7 +73,7 @@ RUN apk add --no-cache --virtual .phpize-deps $PHPIZE_DEPS \
     && yes '' | pecl install -f apcu-5.1.21 \
     && yes '' | pecl install -f imagick-3.7.0 \
     && yes '' | pecl install -f redis-4.3.0 \
-    && yes '' | pecl install -f xdebug-2.9.8 \
+    && yes '' | pecl install -f xdebug-3.1.3 \
```

Comment:
@rocketeerbkw my only question here is whether we should force upgrade people using php7.4 images to Xdebug3 - are we going to mess too much with people's workflows?

Reply:
Potentially yes, because of the port change. Someone who already has xdebug working would need to modify their tool (phpstorm/vscode) to use the new port. Someone who starts using xdebug for the first time after this PR would be fine because tools default to v3 and the new port these days.

If we leave 7.4 with xdebug 2, people who setup xdebug for the first time need to modify their tool ports to the "old" one.